### DISCUSSION TITLE
How do we connect a repository to a project?

### DISCUSSION THREAD

**Discussion by inbalKimchi** [GoldenLabHuji/ElpQuery#20] [General]
How do we link a repository to a project?

**Comment by nadavWeisler** [comment]
(Issue = fancy word for task in GitHub) 
You don't really connect the repository, you connect issues of the repository to your project. The project contains all the issues of all the related repositories.

**Reply by inbalKimchi** [reply]
Thanls! so how do we connect the issues?
